frontstage/templates/sign-in/sign-in.account-locked.html

```diff
@@ -0,0 +1,38 @@
+{% import 'partials/section.html' as section %}
+{% extends "layouts/_twocol.html" %}
+
+{% block page_title %}Trouble signing in? - ONS Business Surveys{% endblock %}
+
+{% block main %}
+    <h1 class="saturn">Check your email</h1>
+
+    <p>You have tried to sign in few times with the wrong details.</p>
+    <p>If your email is registered to an account, we will send an email explaining how to sign in.</p>
```

Comment:
remove `your` does not match content

Reply:
it does . we do not show the email . The card should have been updated. We omit the email adress of the user.